cmd/serve/dlna/dlnaflags/dlnaflags.go

```diff
@@ -45,6 +47,7 @@ func addFlagsPrefix(flagSet *pflag.FlagSet, prefix string, Opt *Options) {
 	flags.StringVarP(flagSet, &Opt.ListenAddr, prefix+"addr", "", Opt.ListenAddr, "The ip:port or :port to bind the DLNA http server to")
 	flags.StringVarP(flagSet, &Opt.FriendlyName, prefix+"name", "", Opt.FriendlyName, "Name of DLNA server")
 	flags.BoolVarP(flagSet, &Opt.LogTrace, prefix+"log-trace", "", Opt.LogTrace, "Enable trace logging of SOAP traffic")
+	flags.StringArrayVarP(flagSet, &Opt.InterfaceNames, prefix+"ifnames", "", Opt.InterfaceNames, "The interfaces to use for SSDP")
```

Comment:
A `StringArray` is so that you can repeat the flag, so `--interface eth0` `--interface eth1`

So `--ifnames` is the wrong name for this flag, it should singular.

I'd suggest `--interface` as being unambigous, so this needs to be something like

    flags.StringArrayVarP(flagSet, &Opt.InterfaceNames, prefix+"interface", "", Opt.InterfaceNames, "The interface to use for SSDP (repeat as necessary)")

Reply:
Ok, good to know will adapt this!